packages/core/src/browser/consoleObservable.ts

```diff
@@ -0,0 +1,96 @@
+import { callMonitored } from '../domain/internalMonitoring'
+import { computeStackTrace } from '../domain/tracekit'
+import { createHandlingStack, ErrorSource, formatErrorMessage, toStackTraceString } from '../tools/error'
+import { mergeObservables, Observable } from '../tools/observable'
+import type { ClocksState } from '../tools/timeUtils'
+import { clocksNow } from '../tools/timeUtils'
+import { find, jsonStringify } from '../tools/utils'
+
+const StatusType = {
+  debug: 'debug',
+  info: 'info',
+  warn: 'warn',
+  error: 'error',
+} as const
+
+type StatusType = typeof StatusType[keyof typeof StatusType]
+
+type ApiNameType = 'log' | 'debug' | 'info' | 'warn' | 'error'
+
+const LogStatusForApi = {
+  log: StatusType.info,
+  debug: StatusType.debug,
+  info: StatusType.info,
+  warn: StatusType.warn,
+  error: StatusType.error,
+}
+
+export interface ConsoleLog {
+  startClocks: ClocksState
```

Comment:
<!-- Thoughts represent an idea that popped up from reviewing. These comments are non-blocking by nature, but they are extremely valuable and can lead to more focused initiatives and mentoring opportunities.  -->
**💭 thought:** ‏it seems weird to me to have a `startClocks` since console calls:

- are punctual events, no difference between start and end
- can't be collected before the SDK init